shared/templates/static/ansible/audit_rules_privileged_commands.yml

```diff
@@ -0,0 +1,65 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Search for privileged commands
+  shell: "find / -xdev -type f -perm -4000 -o -type f -perm -2000 2>/dev/null | cat"
+  check_mode: no
+  register: find_result
+  tags:
+    @ANSIBLE_TAGS@
+
+# Inserts/replaces the rule in /etc/audit/rules.d
```

Comment:
It makes clear which code writes to /etc/audit/rules.d/* and to /etc/audit/audit.rules